README.md

```diff
@@ -59,9 +64,9 @@ You can opt-in for the [beta release on the Play Store](https://play.google.com/
 **Requirements**
 
 * GLib 2.46.2
-* GTK+ 3.12
+* GTK 3.12
 * gcr-3 2.32
-* [WebKit2GTK+](https://webkitgtk.org/) 2.16.6
+* [WebKitGTK](https://webkitgtk.org/) 2.16.6
```

Comment:
The project is called WebKitGTK now, and that's the name used on the project website and blog eg. [WebKitGTK 2.25.4 released!](https://webkitgtk.org/2019/08/02/webkitgtk2.25.4-released.html). GTK as listed here is also `gtk+-3.0` in terms of pkg-config.
The `gcr-3` in this list is actually a mistake, and already here as GCR. I updated the list now for consistency and added websites for all projects except GCR since I couldn't find a dedicated project website.

Reply:
understood :). I wasn't very sure myself to be completely honest